Source/Scene/Cesium3DTile.js

```diff
@@ -297,28 +284,19 @@ define([
         this.selected = false;
```

Comment:
Just to resurface this comment, is this property actually needed? It looks like its still widely used throughout, but does it need to be?

Originally this was added just for the `checkRefiningTiles` stuff that is now gone.